ios/build.sh

```diff
@@ -143,8 +149,9 @@ if [ $BUILD_NUMBER ]; then
   /usr/libexec/Plistbuddy -c "Set CFBundleShortVersionString $BUILD_NUMBER" "$BUILD_FRAMEWORK_PATH/Info.plist"
 fi
 
-rm -r "$APP_FRAMEWORK_PATH"
-cp -R "$BUILD_FRAMEWORK_PATH" "$APP_FRAMEWORK_PATH"
+# Can we get away without this now?
+# rm -r "$APP_FRAMEWORK_PATH"
+# cp -R "$BUILD_FRAMEWORK_PATH" "$APP_FRAMEWORK_PATH"
```

Comment:
About this - the app now directly draws the ios-specific version of the engine framework from its project.

* This negates the need to copy it over 'manually' in the script.
* This prevents the .ipa file from including Simulator-oriented architectures.

When a developer goes to run the Keyman app in the Simulator, as long as they've loaded the joint workspace, Xcode will perform the necessary legwork for the operation will succeed, despite the 'official' build settings.  (It probably creates temporary build settings or something behind the scenes to facilitate this.)